website/docs/foundations/colors/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Colors
 description: This is the description of the colors that will appear in the page cover
-caption: This is the caption of the colors page that will appear in the cards lists
+caption: MD - “Color is used to express style and communicate meaning.”
 ---
```

Comment:
Copied from Material Design just as an example. Do we want something similar to this?

Reply:
I like this approach more than the uber-practical approach of the focus ring example. I'm curious to hear @jorytindall and @cveigt's thoughts, though.

Reply:
Thanks, I'll explore something more along these lines. Will also try to find more examples from other design systems to help with ideas.